console/console-init/ui/src/modules/device/dialogs/CreateDevice/CreateDevice.tsx

```diff
@@ -1,21 +1,174 @@
-import { Wizard } from "@patternfly/react-core";
-import React from "react";
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React, { useState } from "react";
+import {
+  Wizard,
+  WizardFooter,
+  WizardContextConsumer,
+  Button
+} from "@patternfly/react-core";
 import { DeviceInformation } from "modules/device/components/DeviceInformation";
+import { ConnectionType } from "modules/device/components/ConnectionTypeStep";
```

Comment:
can we try to keep consistency with  component 'ConnectionType' name and file name ConnectionTypeStep.
either both ConnectionType or ConnectionTypeStep